GetMessage/handler.ts

```diff
@@ -306,6 +319,35 @@ export function GetMessageHandler(
     }
     const maybeMessageStatus = errorOrMaybeMessageStatus.right;
 
+    const errorOrPaymentStatus = await pipe(
```

Comment:
We must call this API only if:
- The message is ADVANCED
- The caller has the correct permission to see this info
- The message has been processed (pending=false) in order to avoid this call while checking for processing status (i.e: polling)